### PR TITLE
feat(aiven_organization_application_user): SDKv2, add is_super_admin

### PR DESCRIPTION
**Pull request by byashimov**
- Removes beta flag from the user datasource (tech debt)
- Rewrites app users to sdkv2
- Adds `is_super_admin` flag

### CODE REVIEW

**Comment by byashimov** [comment]
[tests](https://github.com/aiven/terraform-provider-aiven/actions/runs/9251982828)

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Small suggestion for the field description to match the Console docs.

**Review by rriski — APPROVED** [review]

File: docs/data-sources/organization_application_user.md [hunk 1]
```diff
@@ -34,5 +29,7 @@ data "aiven_organization_application_user" "tf_user" {
 
 ### Read-Only
 
-- `email` (String) The auto-generated email address of the application user.
+- `email` (String) An email address automatically generated by Aiven to help identify the application user. No notifications are sent to this email.
+- `id` (String) The ID of this resource.
+- `is_super_admin` (Boolean) Alters super admin state of the organization application user.
```
[the remark this answers is not in the document]
Comment (byashimov) [reply docs/data-sources/organization_application_user.md]: Thanks Stacey!

**Review by rriski — APPROVED** [review]

**Review by staceysalamon-aiven — DISMISSED** [review]
LGTM

**Review by rriski — APPROVED** [review]

**Pull request merged**